buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -18,9 +18,14 @@ object Vers {
     // Armeria
```

Comment:
I have predeclared here for future implementation of rate limiting decorator using our jfix-stdlib-ratelimiter.

But I am agree with you that it is better to do not declare unused dependencies since it can confuse other project contributors. 

I will fix it

Reply:
UPD since https://github.com/ru-fix/jfix-armeria/pull/2#discussion_r453700424, I would like to leave jfix-stdlib version declaration :)